crates/astria-bridge-withdrawer/src/bridge_withdrawer/submitter/mod.rs

```diff
@@ -182,25 +176,23 @@ impl Submitter {
         .await
         .context("failed to submit transaction to cometbft")?;
         if let tendermint::abci::Code::Err(check_tx_code) = rsp.check_tx.code {
-            error!(
-                abci.code = check_tx_code,
-                abci.log = rsp.check_tx.log,
-                rollup.height = rollup_height,
-                "transaction failed to be included in the mempool, aborting."
-            );
-            Err(eyre!(
-                "check_tx failure upon submitting transaction to sequencer"
-            ))
+            Err(eyre!(format!(
+                "abci.code = {},
+                abci.log = {},
+                rollup.height = {},
+                check_tx failure upon submitting transaction to sequencer: transaction failed to \
+                 be included in the mempool, aborting.",
+                check_tx_code, rsp.check_tx.log, rollup_height
+            )))
```

Comment:
Not sure I'd try and force the sort of formatting `tracing` produces since this will all end up as a single string in the `message` field I believe.  Also, we don't want line breaks in the message string, and `eyre!` supports passing a format string and args (so need for the `format!` use).
```suggestion
            Err(eyre!(
                "check_tx failure upon submitting transaction to sequencer: transaction failed to \
                 be included in the mempool, aborting. abci.code = {check_tx_code}, abci.log = \
                 {}, rollup.height = {rollup_height}",
                rsp.check_tx.log
            ))
```